app-backend/db/src/main/scala/com/azavea/rf/database/TeamDao.scala

```diff
@@ -51,7 +51,7 @@ object TeamDao extends Dao[Team] {
       fr"""
       SET modified_at = ${now},
           modified_by = ${user.id},
-          name = ${team.name}
+          name = ${team.name},
```

Comment:
_nice_ -- I'm so happy every time we catch one of these things in testing

Reply:
me too!